const e=JSON.parse('{"key":"v-4caeee0c","path":"/java/011001/","title":"例外處理(Exception)","lang":"zh-TW","frontmatter":{"title":"例外處理(Exception)","date":"2021-04-12T16:56:20.000Z","permalink":"/java/011001/","category":["Java","Java基礎","補充內容"],"tag":["程式學習"],"order":1,"description":"Java中的錯誤 Java針對常見錯誤定義了不同的類別這些類別都繼承自 java.lang.Throwable 而Throwable由兩大類所繼承，分別為 java.lang.Exception類別 java.lang.Error類別 Error 只要繼承至java.lang.Error的類別，通常就代表極嚴重的錯誤；此類錯誤發生時，JVM會丟出 Error物件，而這類型的錯誤是無法被當作例外來進行處理的。如： java.lang.OutOfMemoryError","head":[["meta",{"property":"og:url","content":"https://www.ed-lin.com/java/011001/"}],["meta",{"property":"og:site_name","content":"Ed Log 工程人日誌"}],["meta",{"property":"og:title","content":"例外處理(Exception)"}],["meta",{"property":"og:description","content":"Java中的錯誤 Java針對常見錯誤定義了不同的類別這些類別都繼承自 java.lang.Throwable 而Throwable由兩大類所繼承，分別為 java.lang.Exception類別 java.lang.Error類別 Error 只要繼承至java.lang.Error的類別，通常就代表極嚴重的錯誤；此類錯誤發生時，JVM會丟出 Error物件，而這類型的錯誤是無法被當作例外來進行處理的。如： java.lang.OutOfMemoryError"}],["meta",{"property":"og:type","content":"article"}],["meta",{"property":"og:locale","content":"zh-TW"}],["meta",{"property":"og:updated_time","content":"2023-03-02T13:04:26.000Z"}],["meta",{"property":"article:tag","content":"程式學習"}],["meta",{"property":"article:published_time","content":"2021-04-12T16:56:20.000Z"}],["meta",{"property":"article:modified_time","content":"2023-03-02T13:04:26.000Z"}],["script",{"type":"application/ld+json"},"{\\"@context\\":\\"https://schema.org\\",\\"@type\\":\\"Article\\",\\"headline\\":\\"例外處理(Exception)\\",\\"image\\":[\\"\\"],\\"datePublished\\":\\"2021-04-12T16:56:20.000Z\\",\\"dateModified\\":\\"2023-03-02T13:04:26.000Z\\",\\"author\\":[]}"]]},"headers":[{"level":2,"title":"Java中的錯誤","slug":"java中的錯誤","link":"#java中的錯誤","children":[{"level":3,"title":"Error","slug":"error","link":"#error","children":[]}]},{"level":2,"title":"什麼是例外處理","slug":"什麼是例外處理","link":"#什麼是例外處理","children":[]},{"level":2,"title":"例外的引發","slug":"例外的引發","link":"#例外的引發","children":[]},{"level":2,"title":"例外的處理","slug":"例外的處理","link":"#例外的處理","children":[{"level":3,"title":"Throw 敘述","slug":"throw-敘述","link":"#throw-敘述","children":[]},{"level":3,"title":"Try/Catch 敘述","slug":"try-catch-敘述","link":"#try-catch-敘述","children":[]},{"level":3,"title":"Finally 敘述","slug":"finally-敘述","link":"#finally-敘述","children":[]}]}],"git":{"createdTime":1677762266000,"updatedTime":1677762266000,"contributors":[{"name":"LSYu7152","email":"k1207152@gmail.com","commits":1}]},"readingTime":{"minutes":3.08,"words":924},"filePathRelative":"java/01.Java基礎/10.補充內容/01.例外處理(Exception).md","localizedDate":"2021年4月13日","excerpt":"<h2> Java中的錯誤</h2>\\n<p>Java針對常見錯誤定義了不同的類別這些類別都繼承自 <strong>java.lang.Throwable</strong></p>\\n<p>而Throwable由兩大類所繼承，分別為</p>\\n<ul>\\n<li>java.lang.Exception類別</li>\\n<li>java.lang.Error類別</li>\\n</ul>\\n<h3> Error</h3>\\n<p>只要繼承至java.lang.Error的類別，通常就代表極嚴重的錯誤；此類錯誤發生時，JVM會丟出 Error物件，而這類型的錯誤是無法被當作例外來進行處理的。如： <strong>java.lang.OutOfMemoryError</strong></p>","autoDesc":true}');export{e as data};
